The LegalNERo dataset exported to UNER format

pvf2005 opened the pull request.

pvf2005 committed:
Update README.md

pvf2005 committed:
Add LICENSE.txt

pvf2005 committed:
The dataset exported automatically from the LegalNERo corpus

yuvalpinter commented:
@mayhewsw take a look?

mayhewsw commented:
Looks great! Thanks for putting this up. 

Two things:
1. License -- most (some?) of our other data is licensed under [CC BY-SA 4.0](https://creativecommons.org/licenses/by-nc-nd/4.0/deed.en). Specifically, this allows commercial use. Your license does not allow commercial use -- I'd prefer having uniform licenses in the dataset if possible. Is that something you would consider? It's not a blocker if not possible.

2. The format looks good. Our column format allows additional tags in the third column, usually intended for additional or alternate tags. In your case, it would make a lot of sense to include your original tags there.

pvf2005 committed:
Update README.md

pvf2005 committed:
Update LICENSE.txt

pvf2005 committed:
Add files via upload

pvf2005 commented:
I've updated the license for this particular version of the dataset.

Also added some some of the annotations to the other column. However, I don't think all the annotations can be included in there. We have embedded entities within larger entities.
